File: PHP/uberPool.php
<?php
require_once('car.php');

Class UberPool extends Car{

    public $brand;
    public $model;

    public function __construct($license, $brand, $model) {
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }
}

?>

File: Java/UberPool.java
class UberPool extends Car {
    String brand;
    String model;

    public UberPool(String license, Account driver,String brand, String model) {
/*Herencia en java*/ 
        super(license,driver);
        this.brand = brand;
        this.model = model;

        

        
    }
   
}
